client/rest/src/plugins/routeSystem.js

```diff
@@ -44,7 +45,10 @@ const plugins = {
 	namespace,
 	receipts,
 	restrictions,
-	transfer: empty
+	transfer: empty,
+
+	// other
+	rosetta: empty
```

Comment:
Isn't the DB undefined in the empty plugin? Rosetta routes will need access to the db.

Reply:
yes, this will be changed in `rest/dev`